src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -1,6 +1,10 @@
 @import '~amo/css/styles';
 
-.HeroRecommendation {
+.HeroRecommendation-banner {
+  padding-bottom: 0;
```

Comment:
There is another spacing problem somewhere (not here).

Note the spacing between the last shelf and the footer in production:

<img width="921" alt="Screenshot 2019-09-27 12 04 27" src="https://user-images.githubusercontent.com/55398/65787709-2c997100-e11f-11e9-812a-c8faa73776b9.png">


With the `enableFeatureHeroRecommendation` config flag off, I see smaller spacing in your patch:

<img width="921" alt="Screenshot 2019-09-27 12 04 40" src="https://user-images.githubusercontent.com/55398/65787721-3327e880-e11f-11e9-94d2-59f74ac292ec.png">

Reply:
Thanks, I fixed this one as well.

Reply:
Thanks, I fixed this one as well.